### PR TITLE
Update documents with Isaac as the new release manager

### PR DESCRIPTION
**Pull request by ibennetch**
Signed-off-by: Isaac Bennetch bennetch@gmail.com

TODO: 
- [x] @ibennetch should register keybase profile
- [x] Update keyring on fileserver

### CODE REVIEW

File: doc/setup.rst [hunk 1]
```diff
@@ -351,14 +351,25 @@ Verifying phpMyAdmin releases
 +++++++++++++++++++++++++++++
 
 Since July 2015 all phpMyAdmin releases are cryptographically signed by the
-releasing developer, who is currently Marc Delisle. His key id is
+releasing developer, who through January 2015 was Marc Delisle. His key id is
```
Comment (nijel) [inline doc/setup.rst]: It's 2016 now :-)

File: doc/setup.rst [hunk 2]
```diff
@@ -351,14 +351,25 @@ Verifying phpMyAdmin releases
...
 0x81AF644A, his PGP fingerprint is:
 
 .. code-block:: console
 
     436F F188 4B1A 0C3F DCBF 0D79 FEFC 65D1 81AF 644A
 
-and you can get more identification information from `https://keybase.io/lem9 <https://keybase.io/lem9>`_.  You should verify that the signature matches
+and you can get more identification information from `https://keybase.io/lem9 <https://keybase.io/lem9>`_.
+
+Beginning in January 2015, the release manager is Isaac Bennetch. His key id is
```
Comment (nijel) [inline doc/setup.rst]: ...2016

Comment (ibennetch) [inline doc/setup.rst]: I'll fix it, thanks. Good catch.

**Comment by ibennetch** [comment]
Thanks, corrected.

**Comment by nijel** [comment]
Merged, thanks!

**Pull request merged**